.github/workflows/maven.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         distribution: 'temurin'
         cache: maven
     - name: Build with Maven
-      run: mvn -B verify --file pom.xml
+      run: mvn -q -B verify --file pom.xml
```

Comment:
why is -q being added as part of this PR -- is it needed for Lucille as a service? -- and what is the consequence?

Reply:
Trying to quite the 18 Megs of logs during a regular green run ... it didn't work. backing it out